src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/Dec2Hex.txt

```diff
@@ -0,0 +1,142 @@
+﻿﻿// ******** INTEGER PARAMETERS WITHOUT PLACES ********
+
+>> Dec2Hex(0)
+"0"
+
+>> Dec2Hex(Blank())
+"0"
+
+// Positive Integer Parameters
+>> Dec2Hex(165)
+"A5"
+
+>> Dec2Hex(3213)
+"C8D"
+
+>> Dec2Hex(549755813887)
+"7FFFFFFFFF"
+
+// Negative Integer Parameters
+>> Dec2Hex(-1)
+"FFFFFFFFFF"
+
+>> Dec2Hex(-165)
+"FFFFFFFF5B"
+
+>> Dec2Hex(-3213)
+"FFFFFFF373"
+
+>> Dec2Hex(-549755813888)
+"8000000000"
+
+// ******** DECIMAL PARAMETERS ********
+
+>> Dec2Hex(3.74)
+"3"
+
+>> Dec2Hex(-3.74)
+"FFFFFFFFFC"
+
+
+// ******** INTEGER PARAMETERS WITH PLACES ********
+
+>> Dec2Hex(45780)
+"B2D4"
+
+>> Dec2Hex(45780, 3)
+#Error(Kind=InvalidArgument)
+
+>> Dec2Hex(45780, 4)
+"B2D4"
+
+>> Dec2Hex(45780, 5)
+"0B2D4"
+
+>> Dec2Hex(45780, 6)
+"00B2D4"
+
+>> Dec2Hex(45780, -6)
+#Error(Kind=Numeric)
+
+>> Dec2Hex(45780, 11)
+#Error(Kind=Numeric)
+
+// Places are ignored when number is negative
+>> Dec2Hex(-45780)
+"FFFFFF4D2C"
+
+>> Dec2Hex(-45780, 4)
+"FFFFFF4D2C"
+
+>> Dec2Hex(-45780, 5)
+"FFFFFF4D2C"
+
+>> Dec2Hex(-45780, 6)
+"FFFFFF4D2C"
+
+>> Dec2Hex(-45780, 11)
+#Error(Kind=Numeric)
+
+// ******** NULL and ERROR PARAMETERS ********
+>> Dec2Hex(1/0)
+#Error(Kind=Div0)
+
+>> Dec2Hex(-549755813889)
+#Error(Kind=Numeric)
+
+>> Dec2Hex(549755813888)
+#Error(Kind=Numeric)
+
+// ******** STRING PARAMETERS ********
+
+>> Dec2Hex("sampleText")
+#Error(Kind=InvalidArgument)
+
+>> Dec2Hex("4")
+"4"
+
+>> Dec2Hex("14", 4)
+"000E"
+
+>> Dec2Hex("15", "6")
+"00000F"
+
+>> Dec2Hex("-9")
+"FFFFFFFFF7"
+
+// ******** BOOLEAN PARAMETERS ********
+
+>> Dec2Hex(true)
+"1"
+
+>> Dec2Hex(false)
+"0"
+
+>> Dec2Hex("true")
+#Error(Kind=InvalidArgument)
+
+>> Dec2Hex("false")
+#Error(Kind=InvalidArgument)
+
+// ******** FUNCTION VALUE AS PARAMETERS ********
+
+>> Dec2Hex(Ln(2))
+"0"
+
+>> Dec2Hex(Power(2,2))
+"4"
+
+>> Dec2Hex([10,255,-16])
+["A","FF","FFFFFFFFF0"]
+
+>> Dec2Hex([1,1e-45,1e45,2])
+["1","0",Microsoft.PowerFx.Types.ErrorValue,"2"]
+
+>> Dec2Hex([10,11,"12"], [2,5,10])
+["0A","0000B","000000000C"]
+
+>> Dec2Hex([10,255,-16], 5)
+["0000A","000FF","FFFFFFFFF0"]
+
+>> Dec2Hex(10, [2,5,10])
+["0A","0000A","000000000A"]
```

Comment:
>] [](http://example.com/codeflow?start=26&length=1)

More tests with blanks and errors inside tables... Take a look at \src\tests\Microsoft.PowerFx.Core.Tests\ExpressionTestCases\TabularOverloadsBlanksAndErrors.txt and add some tests in that same pattern...